core/lib/generators/spree/install/install_generator.rb

```diff
@@ -4,6 +4,8 @@
 
 module Spree
   class InstallGenerator < Rails::Generators::Base
+    SPREE_MOUNT_ROUTE = "mount Spree::Core::Engine"
```

Comment:
This is already namespaced within `Spree::`, so I'm not sure that's descriptive. It could be `MOUNT_ROUTE` or `ENGINE_MOUNT_ROUTE` or `CORE_MOUNT_ROUTE`

Reply:
changed